webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx

```diff
@@ -435,6 +444,18 @@ const TableActionBar: React.FC<Props> = ({
         sourceWorkspaceId={project.workspaceId}
         onSubmit={handleSubmitMove}
       />
+      <ExperimentRetainLogsModal.Component
+        excludedExperimentIds={excludedExperimentIds}
+        experimentIds={experimentIds.filter(
+          (id) =>
+            canActionExperiment(ExperimentAction.RetainLogs, experimentMap[id]) &&
+            permissions.canModifyExperiment({
+              workspace: { id: experimentMap[id].workspaceId },
+            }),
+        )}
+        filters={selectAll ? filters : undefined}
+        onSubmit={handleSubmitMove}
```

Comment:
qq: do you want `handleSubmitMove` for `ExperimentRetainLogsModal`? whats this for?

Reply:
We needed handleSubmitRetain logs. Fixed it!